src/vs/workbench/services/authentication/browser/authenticationService.ts

```diff
@@ -567,7 +567,9 @@ export class AuthenticationService extends Disposable implements IAuthentication
 					key: 'accessRequest',
 					comment: ['The placeholder {0} will be replaced with an extension name. (1) is to indicate that this menu item contributes to a badge count']
```

Comment:
it would be good to also update the comment for the people localizing the string

Reply:
oops! Good point.